web.py

```diff
@@ -230,39 +262,13 @@ async def pairs(session=Depends(get_db)):
                 "quote_id": quote.hash,
                 "quote_name": quote.name,
                 "quote_symbol": quote.symbol,
-                "last_price": quote_price or last_price,
-                "base_volume": base_volume or 0,
-                "quote_volume": quote_volume or 0,
+                "last_price": FormattedFloat(quote_price or last_price),
+                "base_volume": FormattedFloat(base_volume or 0) or 0,
+                "quote_volume": FormattedFloat(quote_volume or 0) or 0,
             }
-    return pairs
-
-
-class PairResponse(BaseModel):
-    """
-    Definition of specific pair endpoint response format.
-    Used to generate docs.
-    """
-
-    base_id: str = Field(
-        "0x0200040000000000000000000000000000000000000000000000000000000000"
-    )
-    base_name: str = Field("SORA Validator Token")
-    base_symbol: str = Field("VAL")
-    quote_id: str = Field(
-        "0x0200000000000000000000000000000000000000000000000000000000000000"
-    )
-    quote_name: str = Field("SORA")
-    quote_symbol: str = Field(example="XOR")
-    last_txid: str = Field(
-        example="0x1234000000000000000000000000000000000000000000000000000000000000"
-    )
-    last_block: int = Field(example=100)
-    last_price: float = Field(example=12.34)
-    base_volume: float = Field(example=5.6)
-    quote_volume: float = Field(example=7.8)
-
+    return FormattedJSONResponse(pairs)
 
-@app.get("/pairs/{base}-{quote}/", response_model=PairResponse)
+@app.get("/pairs/{base}-{quote}/")
```

Comment:
Removing response_model would break the documentation and the swagger:
https://stats.sora.org/docs#/default/pair_pairs__base___quote___get
@takemiyamakoto We couldn't find a better way to implement it. Do you know if anybody uses our documentation or swagger?